ako-operator/controllers/statefulset.go

```diff
@@ -169,11 +169,12 @@ func BuildStatefulSet(ako akov1alpha1.AKOConfig, aviSecret corev1.Secret) (appsv
 	if err != nil {
 		return sf, err
 	}
-	var replicas int32 = 1
+	replicas := int32(ako.Spec.ReplicaCount)
```

Comment:
I think we have to limit the `replicaCount` to 2 always.

Reply:
Added check to limit replica count to 2